test/lib/model/test_classycat.py

```diff
@@ -562,9 +563,17 @@ def test_classify_fail_wrong_response_format(self, file_exists_in_s3_mock, uploa
             }
         }
         classify_message = schemas.parse_message(classify_input)
-        result = self.classycat_model.process(classify_message)
 
-        self.assertEqual(result.responseMessage, "Error classifying items: list index out of range")
+        try:
+            result = self.classycat_model.process(classify_message)
```

Comment:
side comment:  there is an assertRaises pattern in the unitTest framework that can be helpful for these

Reply:
great idea, updated